src/main.js

```diff
@@ -8,12 +11,23 @@ var savedCovers = [
 var currentCover;
 
 // Add your event listeners here 👇
-
+window.onload = displayMainPageCover;
 
 // Create your event handlers and other functions here 👇
 
 
 // We've provided one function to get you started
+function displayMainPageCover() {
+  var randomCover = covers[getRandomIndex(covers)];
+  var randomTitle = titles[getRandomIndex(titles)];
+  var randomDescriptor = descriptors[getRandomIndex(descriptors)];
+  var randomDescriptor2 = descriptors[getRandomIndex(descriptors)];
+  coverImage.setAttribute('src', randomCover);
+  coverTitle.innerHTML = randomTitle;
+  discriptorTagLine1.innerText = randomDescriptor;
```

Comment:
We might consider using just one method here.

Reply:
Let's look into refining this.